docs/source/_static/environment.yml

```diff
@@ -0,0 +1,28 @@
+name: apple_tensorflow
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - absl-py
+  - astunparse
+  - gast
+  - google-pasta
+  - grpcio
+  - h5py
+  - ipython
+  - keras-preprocessing
+  - numpy
+  - opt_einsum
+  - pip=20.2.4
+  - protobuf
+  - python-flatbuffers
+  - python=3.8
```

Comment:
What about python 3.9 ?

Reply:
Again, this probably works but was written before. Although, I am unsure if their TF version is completely compatible. Again, I did not write this file so I do not know their reasoning.